packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.m

```diff
@@ -23,5 +23,6 @@ @interface RCT_EXTERN_MODULE(RNReactNativeGutenbergBridge, NSObject)
 RCT_EXTERN_METHOD(addMention:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)rejecter)
 RCT_EXTERN_METHOD(requestStarterPageTemplatesTooltipShown:(RCTResponseSenderBlock)callback)
 RCT_EXTERN_METHOD(setStarterPageTemplatesTooltipShown:(BOOL)tooltipShown)
+RCT_EXTERN_METHOD(actionButtonPressed:(NSString *)buttonType)
```

Comment:
Given that `RNReactNativeGutenbergBridge` does many things, it's not clear from the name that `actionButtonPressed` refers to a button on the action sheet that is shown on unsupported blocks. 
Could we change this to something like `unsupportedBlockActionButtonPressed`?

Reply:
The idea of this is to be generic, and to be able to use it with other action buttons that doesn't need any particular data.

Reply:
Gotcha